helm/vernemq/README.md

```diff
@@ -104,8 +104,9 @@ Parameter | Description | Default
 `statefulset.podManagementPolicy` | start and stop pods in Parallel or OrderedReady (one-by-one.)  **Note** - Cannot change after first release. | `OrderedReady`
 `statefulset.terminationGracePeriodSeconds` | configure how much time VerneMQ takes to move offline queues to other nodes | `60`
 `statefulset.updateStrategy` | Statefulset updateStrategy | `RollingUpdate`
-`statefulset.lifecycle | Statefulset lifecycle hooks | `{}`
+`statefulset.lifecycle` | Statefulset lifecycle hooks | `{}`
 `serviceMonitor.create` | whether to create a ServiceMonitor for Prometheus Operator | `false`
+`serviceMonitor.labels` | whether to add omre labels to ServiceMonitor for Prometheus Operator | `{}`
```

Comment:
Hey @ofirshtrull, I noticed a small typo in the description. Please change `omre` -> `more`

Reply:
Fixed